zerver/migrations/0090_userprofile_high_contrast_mode.py

```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('zerver', '0089_auto_20170710_1353'),
+        ('zerver', '0089_user_soft_deactivation'),
```

Comment:
We can't rename this migration once it's been released; the database keeps track of not just the number but also the name.